app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+
+  def show
+    @user = User.find_by id:params[:id]
+  end 
```

Comment:
```suggestion
  before_action :load_user, only: :show
  
  def show; end
  
  private
  
  def load_user
        ...
  end
```
You create a private method to test the existence of @user. Then create a before_action for the action 'show'.